fili-core/src/main/java/com/yahoo/bard/webservice/web/RateLimitCleanupOnRequestComplete.java

```diff
@@ -0,0 +1,15 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
```

Comment:
push to sub package.  This is an implementation detail.

Reply:
done